### ISSUE TITLE
svelte-konva, is it ready ?

### ISSUE THREAD

**Issue by projektorius96** [konvajs/konva#1488]
Couple of months ago I've initiated svelte-konva project currently [svelte-konva__archived#v1.1.0](https://github.com/projektorius96/svelte-konva__archived/releases/tag/v1.1.0) I am not sure how mature this project looks or could be and has any potential, but as I continue using Svelte rather than React or Vue, I thought some Svelte consumers might be interested into contributing, or at least giving some remarks maintaining the project in the future ...

Tagging @lavrton as well .

Thank you .

**Comment by lavrton** [comment]
Hello @projektorius96. That looks very promising to me! I even would like to have it as official plugin in "konva tools family". It will be really good if you can polish README file with installation example and make several online demos (with codesandbox probably).

**Comment by projektorius96** [comment]
> Hello @projektorius96. That looks very promising to me! I even would like to have it as official plugin in "konva tools family". It will be really good if you can polish README file with installation example and make several online demos (with codesandbox probably).

Definitely agree that `README.md` shall be polished, be more legible in general ! I'll try to find more inspiration working on demos as well as code readability itself . 

Much appreciated ! @lavrton

**Comment by projektorius96** [comment]
@lavrton coming back with more precise fully-fledged repo example [svelte-konva](https://github.com/projektorius96/svelte-konva/releases/tag/v0.0.1) ; <br>

[demo on REPLIT](https://replit.com/@projektorius96/svelte-konva-demo)

---

- I'd like to give a huge remark to @TeyKey1 as author of alternative [svelte-konva](https://github.com/TeyKey1/svelte-konva) project which seems to give more Svelte alike approach, wherein the codebase seems to be based on "from-scratch" paradigm following HTML-like mark-up, just like react-konva or vue-svelte by @lavrton himself .

- My approach serves solely as "svelte-for-konva wrapper" solution based on minimum `../utils/ComponentLinker.js` logic wherein we take things out of `.$$.ctx` internals `.at(0)` and hook it as if done via precise Component API, keep in mind that using default slots in Svelte approach of mine may not work as expected, unless explicitly using Component API which it has its own nuances doing it !

I am not closing this yet, waiting for @lavrton response. Cheers !

**Comment by TeyKey1** [comment]
Hello everyone 

@projektorius96 already described my svelte-konva wrapper pretty well. To add some perspective: Originally, I was using Konva heavily in combination with vue-konva. As I later switched to Svelte, I decided to create a Svelte wrapper library which should be as similar as possible to vue-konva. While building the wrapper, I realized there was already an alternative available ([svelte-konvajs](https://github.com/PatWie/svelte-konvajs)) which is architecturally very similar to what I built (and still is). As svelte-konvajs does not seem to be maintained anymore, I decided to just push on and finish/release my library.

In terms of usage, the library is very similar to vue/react-konva and also fully component-based. It adds some special Svelte functionality to the mix, for example by supporting Svelte bindings. The library is currently in a production-ready state, reasonably well tested and documented. As such, it is already available on the npm package registry.

I'll let you decide on how to further proceed with this. Personally, I'd be willing to integrate my wrapper into the Konva ecosystem and keep maintaining it there, if desired. If some other Svelte wrapper alternatives are considered to be moved into the Konva repo, we can surely transfer the current "svelte-konva" name registered on npm to be used by the official Svelte konva wrapper to keep the naming of the official wrappers consistent.
